<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>526</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>20</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
 background-color: #024873;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="result_show">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>301</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>26</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
  qproperty-alignment: 'AlignVCenter | AlignRight';
  background-color: #04BFAD;
  color:#FFFFFF; 
  border: 1px solid gray;
  border-radius: 20px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_in">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>52</width>
      <height>52</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Open Sans</family>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	color: white;
   border: 1px solid gray;
   background-color: #0889A6;
   border-radius:25px;
   max-width:50px;
   max-height:50px;
   min-width:50px;
  min-height:50px;

}
QPushButton:pressed {
    background-color: #F7E425;
}</string>
    </property>
    <property name="text">
     <string>( )</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_change">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>170</y>
      <width>52</width>
      <height>52</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Open Sans</family>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	color: white;
   border: 1px solid gray;
   background-color: #0889A6;
   border-radius:25px;
   max-width:50px;
   max-height:50px;
   min-width:50px;
  min-height:50px;

}
QPushButton:pressed {
    background-color: #F7E425;
}</string>
    </property>
    <property name="text">
     <string>+/-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_percent">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>170</y>
      <width>52</width>
      <height>52</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Open Sans</family>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	color: white;
   border: 1px solid gray;
   background-color: #0889A6;
   border-radius:25px;
   max-width:50px;
   max-height:50px;
   min-width:50px;
  min-height:50px;

}
QPushButton:pressed {
    background-color: #F7E425;
}</string>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_division">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>170</y>
      <width>52</width>
      <height>52</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Open Sans</family>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	color: white;
   border: 1px solid gray;
   background-color: #0889A6;
   border-radius:25px;
   max-width:50px;
   max-height:50px;
   min-width:50px;
  min-height:50px;

}
QPushButton:pressed {
    background-color: #F7E425;
}</string>
    </property>
    <property name="text">
     <string>รท</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_mult">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>240</y>
      <width>52</width>
      <height>52</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Open Sans</family>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	color: white;
   border: 1px solid gray;
   background-color: #0889A6;
   border-radius:25px;
   max-width:50px;
   max-height:50px;
   min-width:50px;
  min-height:50px;

}
QPushButton:pressed {
    background-color: #F7E425;
}</string>
    </property>
    <property name="text">
     <string>ร</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>240</y>
      <width>52</width>
      <height>52</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Open Sans</family>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   color: #024873;
   border: 1px solid gray;
   background-color: #B0D1D8;
   border-radius:25px;
   max-width:50px;
   max-height:50px;
   min-width:50px;
  min-height:50px;

}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #04BFAD, stop: 1 #04BFAD);
}</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>52</width>
      <height>52</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Open Sans</family>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   color: #024873;
   border: 1px solid gray;
   background-color: #B0D1D8;
   border-radius:25px;
   max-width:50px;
   max-height:50px;
   min-width:50px;
  min-height:50px;

}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #04BFAD, stop: 1 #04BFAD);
}</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>240</y>
      <width>52</width>
      <height>52</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Open Sans</family>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   color: #024873;
   border: 1px solid gray;
   background-color: #B0D1D8;
   border-radius:25px;
   max-width:50px;
   max-height:50px;
   min-width:50px;
  min-height:50px;

}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #04BFAD, stop: 1 #04BFAD);
}</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sub">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>310</y>
      <width>52</width>
      <height>52</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Open Sans</family>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	color: white;
   border: 1px solid gray;
   background-color: #0889A6;
   border-radius:25px;
   max-width:50px;
   max-height:50px;
   min-width:50px;
  min-height:50px;

}
QPushButton:pressed {
    background-color: #F7E425;
}</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>310</y>
      <width>52</width>
      <height>52</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Open Sans</family>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   color: #024873;
   border: 1px solid gray;
   background-color: #B0D1D8;
   border-radius:25px;
   max-width:50px;
   max-height:50px;
   min-width:50px;
  min-height:50px;

}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #04BFAD, stop: 1 #04BFAD);
}</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>52</width>
      <height>52</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Open Sans</family>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   color: #024873;
   border: 1px solid gray;
   background-color: #B0D1D8;
   border-radius:25px;
   max-width:50px;
   max-height:50px;
   min-width:50px;
  min-height:50px;

}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #04BFAD, stop: 1 #04BFAD);
}</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>310</y>
      <width>52</width>
      <height>52</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Open Sans</family>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   color: #024873;
   border: 1px solid gray;
   background-color: #B0D1D8;
   border-radius:25px;
   max-width:50px;
   max-height:50px;
   min-width:50px;
  min-height:50px;

}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #04BFAD, stop: 1 #04BFAD);
}</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_add">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>380</y>
      <width>52</width>
      <height>52</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Open Sans</family>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	color: white;
   border: 1px solid gray;
   background-color: #0889A6;
   border-radius:25px;
   max-width:50px;
   max-height:50px;
   min-width:50px;
  min-height:50px;

}
QPushButton:pressed {
    background-color: #F7E425;
}</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>380</y>
      <width>52</width>
      <height>52</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Open Sans</family>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   color: #024873;
   border: 1px solid gray;
   background-color: #B0D1D8;
   border-radius:25px;
   max-width:50px;
   max-height:50px;
   min-width:50px;
  min-height:50px;

}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #04BFAD, stop: 1 #04BFAD);
}</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>380</y>
      <width>52</width>
      <height>52</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Open Sans</family>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   color: #024873;
   border: 1px solid gray;
   background-color: #B0D1D8;
   border-radius:25px;
   max-width:50px;
   max-height:50px;
   min-width:50px;
  min-height:50px;

}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #04BFAD, stop: 1 #04BFAD);
}</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>380</y>
      <width>52</width>
      <height>52</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Open Sans</family>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   color: #024873;
   border: 1px solid gray;
   background-color: #B0D1D8;
   border-radius:25px;
   max-width:50px;
   max-height:50px;
   min-width:50px;
  min-height:50px;

}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #04BFAD, stop: 1 #04BFAD);
}</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_equal">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>450</y>
      <width>52</width>
      <height>52</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Open Sans</family>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	color: white;
   border: 1px solid gray;
   background-color: #0889A6;
   border-radius:25px;
   max-width:50px;
   max-height:50px;
   min-width:50px;
  min-height:50px;
}
QPushButton:pressed {
    background-color: #F7E425;
}</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_0">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>450</y>
      <width>52</width>
      <height>52</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Open Sans</family>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   color: #024873;
   border: 1px solid gray;
   background-color: #B0D1D8;
   border-radius:25px;
   max-width:50px;
   max-height:50px;
   min-width:50px;
  min-height:50px;

}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #04BFAD, stop: 1 #04BFAD);
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_C">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>450</y>
      <width>52</width>
      <height>52</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Open Sans</family>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	color: white;
   border: 1px solid gray;
   background-color: #F79797;
   border-radius:25px;
   max-width:50px;
   max-height:50px;
   min-width:50px;
  min-height:50px;

}
QPushButton:pressed {
    background-color: #F25E5E;
}</string>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_dot">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>450</y>
      <width>52</width>
      <height>52</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Open Sans</family>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   color: #024873;
   border: 1px solid gray;
   background-color: #B0D1D8;
   border-radius:25px;
   max-width:50px;
   max-height:50px;
   min-width:50px;
  min-height:50px;

}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #04BFAD, stop: 1 #04BFAD);
}</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QLabel" name="result_show_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>301</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
  qproperty-alignment: 'AlignVCenter | AlignRight';
  background-color: #04BFAD;
  color:#FFFFFF; 
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
